tests/test_routes.py

```diff
@@ -334,6 +339,77 @@ def test_method_not_allowed(self):
+    # T E S T   Q U E R Y   S T R I N G S
+######################################################################
+
+    def test_query_inventories_by_name(self):
+        """It should Query Inventories by Name"""
+        records = self._create_inventory_records(10)
+        test_name = records[0].name
+        name_list = [record for record in records if record.name == test_name]
+        logging.info("Name=%s: %d = %s", test_name, len(name_list), name_list)
+        resp = self.client.get(BASE_URL, query_string=f"name={quote_plus(test_name)}")
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        data = resp.get_json()
+        self.assertEqual(len(data), len(name_list))
+        # check the data just to be sure
+        for pet in data:
+            self.assertEqual(pet["name"], test_name)
+
+    def test_query_inventories_by_condition(self):
+        """It should Query Inventories by Condition"""
+        records = self._create_inventory_records(1)
+        test_condition = records[0].condition
+        condition_list = [record for record in records if record.condition.name == test_condition.name]
+        logging.info(
+            "Category=%s: %d = %s", test_condition, len(condition_list), condition_list
+        )
+        resp = self.client.get(
+            BASE_URL, query_string=f"condition={quote_plus(test_condition.value)}"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        data = resp.get_json()
+        self.assertEqual(len(data), len(condition_list))
+        # check the data just to be sure
+        for record in data:
+            self.assertEqual(record["condition"], test_condition.value)
+
+    def test_query_inventories_by_active(self):
+        """It should Query Pets by Availability"""
+        records = self._create_inventory_records(10)
+        test_active = records[0].active
+        active_list = [record for record in records if record.active == test_active]
+        logging.info(
+            "Active=%s: %d = %s", test_active, len(active_list), active_list
+        )
+        resp = self.client.get(BASE_URL, query_string=f"active={str(test_active)}")
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        data = resp.get_json()
+        self.assertEqual(len(data), len(active_list))
+        # check the data just to be sure
+        for record in data:
+            self.assertEqual(record["active"], test_active)
+
+    def test_query_pets_by_quantity(self):
+        """It should Query Inventories by Quantity"""
+        records = self._create_inventory_records(10)
+        test_quantity = records[0].quantity
+        quantity_list = [record for record in records if record.quantity == test_quantity]
+        logging.info(
+            "Quantity=%s: %d = %s", test_quantity, len(quantity_list), quantity_list
+        )
+        resp = self.client.get(BASE_URL, query_string=f"quantity={str(test_quantity)}")
```

Comment:
Looks like `f"quantity={test_quantity}"` instead. Same above.

Reply:
yeah I can change this

Reply:
done